telemetry/client.go

```diff
@@ -413,3 +450,26 @@ func (c *Client) UpdateEnvelopeProcessingError(shhMessage *types.Message, proces
 		c.logger.Error("Error sending envelope update to telemetry server", zap.Error(err))
 	}
 }
+
+func (c *Client) ProcessReuglarStoryRetrievedMsgs(data MetricPayload) {
+	fmt.Println(data)
+
+	postBody := map[string]interface{}{
```

Comment:
One thing to consider is to drop any processing on client and just push the metric as it is structured by Prometheus to the Telemetry server - then we can do whatever we want/need with it on the server - simplifies client quite a lot